qemu/tests/virtio_port_hotplug.py

```diff
@@ -67,7 +68,10 @@ def get_virtio_port_name_by_params(params, tag):
             if not virtio_port:
                 test.fail("Virtio Port named '%s' not found" % port_name)
             chardev_qid = virtio_port.get_param("chardev")
-            port_chardev = vm.devices.get_by_qid(chardev_qid)[0]
+            try:
+                port_chardev = vm.devices.get_by_qid(chardev_qid)[0]
+            except IndexError:
+                raise DeviceError("Failed to get chardev %s's port !" % chardev_qid)
```

Comment:
You don't need `DeviceError`, `test.error()` is sufficient. And it's not `chardev %s's port`, you might say: `fail to get device %s' % chardev_qid`

Reply:
Thanks and updated.